src/appmetrics.cpp

```diff
@@ -388,10 +397,22 @@ static void sendData(const std::string &sourceId, unsigned int size, void *data)
 	memcpy(dataCopy, data, size);
 	payload->data = dataCopy;
 	payload->size = size;
-
-	async->data = payload;
-	uv_async_init(uv_default_loop(), async, (uv_async_cb)emitMessage);
```

Comment:
Looks like `async` is no longer used, but it is still declared (https://github.com/RuntimeTools/appmetrics/pull/185/files#diff-2419e406d80ff8bfd9b4fc942cc6b491R392)